class.jetpack.php

```diff
@@ -2817,9 +2817,13 @@ public static function get_stat_data( $encode = true ) {
 	 * Get additional stat data to sync to WPCOM
 	 */
 	public static function get_additional_stat_data( $prefix = '' ) {
+		global $wpdb;
+		$id = get_current_blog_id();
+		$blog_prefix = $wpdb->get_blog_prefix( $id );
+
 		$return["{$prefix}themes"]         = Jetpack::get_parsed_theme_data();
 		$return["{$prefix}plugins-extra"]  = Jetpack::get_parsed_plugin_data();
-		$return["{$prefix}users"]          = count_users();
+		$return["{$prefix}users"]          = $wpdb->get_var( "SELECT COUNT(USER_ID) FROM $wpdb->usermeta WHERE meta_key = '{$blog_prefix}capabilities'" );
```

Comment:
Can't we just use `$wpdb->prefix` instead of getting `$blog_prefix` with `$wpdb->get_blog_prefix( $id );`.